CNV/cnv-2.1.0.sh

```diff
@@ -154,3 +154,52 @@ metadata:
   namespace: "${TARGET_NAMESPACE}"
 spec: {}
 EOF
+
+echo "Waiting for HCO to get fully deployed"
+oc wait -n ${TARGET_NAMESPACE} hyperconverged hyperconverged-cluster --for condition=Available --timeout=10m
+
+echo "Setting up a bridge as the default interface"
+machineCIDR=$(grep 'machineCIDR' ../OpenShift/install-config.yaml | sed 's/\(.*\): *\(.*\)/\2/')
+while ! default_iface=$(oc get nodenetworkstate -o jsonpath="{.items[0].status.currentState.routes.running[?(@.destination==\"${machineCIDR}\")].next-hop-interface}"); do
```

Comment:
Do we have to be so smart? Isn't it known that `default_iface==eno1`?
I worry that this code would break in case `brext` already exists on top of `eno1`.

(Don't get me wrong: at some point we'd want a smart Policy of "apply this on the nic that is on the default route and keep the existing IP")

Reply:
Unfortunately we need this. On bare-metal env I used the default nic was eno4, on virtual ens3 (or something similar), it is not even always the first found NIC. This script has two assumptions.

* All nodes are the same, have the same nic connected to the default network
* This script is called only once during the initial installation (we can improve it and just skip the config in case brext is already default)